src/coreclr/jit/llvm.cpp

```diff
@@ -656,6 +656,17 @@ Type* Llvm::getLlvmTypeForVarType(var_types type)
     }
 }
 
+Type* Llvm::getLlvmTypeForLclVar(GenTreeLclVarCommon* lclVar)
+{
+    var_types varType = lclVar->TypeGet();
```

Comment:
```suggestion
    var_types nodeType = lclVar->TypeGet();
```
Nit: the `varType` strongly implies we're dealing with `varDsc->TypeGet()` but actually that's not the case (they will be different for `LCL_FLD`s).